components/modal/Register/register.component.js

```diff
@@ -1,214 +1,216 @@
-import React, { Component } from 'react';
-import { Form, Input, Label, FormGroup, FormFeedback } from 'reactstrap';
-import { isEmail, isDecimal } from 'validator';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { isEmail } from 'validator';
+import PropTypes from 'prop-types';
 import classNames from './register.component.module.scss';
-class Register extends Component {
 
-    constructor(props) {
-        super(props);
-        this.state = this.getInitialState(props);
+function Register(props) {
+  const { register, handleSubmit, errors } = useForm();
+  const getMembersIntroURL = (RDSID) => `https://api.realdevsquad.com/members/intro/${RDSID}`;
+  const parameters = props.rdsusername;
+  const onSubmit = (data) => {
+    console.log(data);
+    function charRem(paramstr) {
```

Comment:
What is `charRem` here? 
Please give a proper self-explanatory name.

Reply:
The string that comes through the props is like '@username'. So we cant make an API call with @ in it. Thus, it filters out the @ symbol and returns the username, appropriate for the API URL. Regardless of that, I shall give it a better name.